W5500.device.nut

```diff
@@ -1356,6 +1359,13 @@ class W5500.Driver {
     function sendSocketCommand(socket, command) {
         local bsb = _getSocketRegBlockSelectBit(socket);
         writeReg(W5500_SOCKET_COMMAND, bsb, command);
+        local started = hardware.millis();
+        while (readReg(W5500_SOCKET_COMMAND, bsb) != 0x00) {
+            // Check for timeouts here
+            if (hardware.millis() - started > (W5500_COMMAND_TIMEOUT * 1000)) throw W5500_ERR_COMMAND_TIMEOUT;
```

Comment:
I would probably just do `server.error` here instead. Otherwise restarting the device because of a Wiznet issue doesn't sound like a good idea. Or we should clearly document that sending a message may throw a timeout error that needs to be handled gracefully.